src/Symfony/Component/Cache/Traits/PhpFilesTrait.php

```diff
@@ -32,6 +32,35 @@ public static function isSupported()
     }
 
     /**
+     * @return bool
+     */
+    public function prune()
+    {
+        $time = time();
+        $pruned = true;
+        $allowCompile = static::opcacheAllowCompile();
+
+        set_error_handler($this->includeHandler);
+        try {
+            foreach (new \RecursiveIteratorIterator(new \RecursiveDirectoryIterator($this->directory, \FilesystemIterator::SKIP_DOTS), \RecursiveIteratorIterator::LEAVES_ONLY, \RecursiveIteratorIterator::CATCH_GET_CHILD) as $file) {
```

Comment:
you're sending back my questions above :)
if I can decide: it's not used right now + nobody complained => let's continue to not use it at all